src/04-modality-specific-files/03-electroencephalography.md

```diff
@@ -376,7 +376,7 @@ Fields relating to the EEG electrode positions:
 | Keyword                        | Description                                                                                                                                                            |
 | --------------------------------------------------------------------------------------------------------------------------------------------------| --------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | EEGCoordinateSystem            | REQUIRED. Refers to the coordinate system in which the EEG electrode positions are to be interpreted (see [Appendix VIII](../99-appendices/08-coordinate-systems.md)). |
-| EEGCoordinateUnits             | REQUIRED. Units in which the coordinates that are listed in the field `EEGCoordinateSystem` are represented (e.g., "mm", "cm").                                        |
+| EEGCoordinateUnits             | REQUIRED. Units in which the coordinates that are listed in the field `EEGCoordinateSystem` are represented. MUST be `m`, `cm`, or `mm`.                               |
```

Comment:
I see this change as a clarification / issue fix, because the EEG spec were to follow the MEG spec in this regard, and MEG clearly states: 

> MUST be `m`, `cm`, or `mm`

[see coords section](https://bids-specification.readthedocs.io/en/latest/04-modality-specific-files/02-magnetoencephalography.html#coordinate-system-json-_coordsystemjson)

This change is also applied to `FiducialsCoordinateUnits` and `AnatomicalLandmarkCoordinateUnits`

yet ... in the MRI [`AnatomicalLandmarkCoordinateUnits` section](https://bids-specification.readthedocs.io/en/latest/04-modality-specific-files/01-magnetic-resonance-imaging-data.html#anatomical-landmarks), voxels are given as the unit to be used --> So I wonder whether we should add `voxels` as permitted in EEG and MEG as well

allowing `voxels` on top of m, cm, and mm would also correspond nicely to the allowed `pixels` unit in iEEG for these fields.